docs/caveats.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Caveats
+---
+
+## Feature support
+
+| Feature               | Lua 5.1 | Lua 5.2 | Lua 5.3 | LuaJIT |
+| --------------------- | :-----: | :-----: | :-----: | :----: |
+| (Everything else)     |   ✔️    |   ✔️    |   ✔️    |   ✔️   |
+| [Bitwise operators]   |   ❌    |   ✔️    |   ✔️    |   ✔️   |
+| [Switch statement]    |   ❌    |   ✔️    |   ✔️    |   ✔️   |
+| [`continue`]          |   ❌    |   ✔️    |   ✔️    |   ✔️   |
+| [`Promise`]           |   ❌    |   ❌    |   ❌    |   ❌   |
+| [`async`] / [`await`] |   ❌    |   ❌    |   ❌    |   ❌   |
+| [Regular Expressions] |   ❌    |   ❌    |   ❌    |   ❌   |
+| [Optional Chaining]   |   ❌    |   ❌    |   ❌    |   ❌   |
+| [Nullish Coalescing]  |   ❌    |   ❌    |   ❌    |   ❌   |
+| [Private Fields]      |   ❌    |   ❌    |   ❌    |   ❌   |
+| [JSX]                 |   ❌    |   ❌    |   ❌    |   ❌   |
```

Comment:
TSX?

Reply:
The feature itself is still called JSX in [docs](https://www.typescriptlang.org/docs/handbook/jsx.html), options and compiler code. `tsx` is used only as an extension